policy/policy.go

```diff
@@ -63,6 +63,33 @@ type UpdatedList struct {
 	list
 }
 
+// DomainsToValidate [interface Validator] retrieves domains from the
+// DB whose policies should be validated.
+func (l UpdatedList) DomainsToValidate() ([]string, error) {
+	l.mu.RLock()
+	defer l.mu.RUnlock()
+	domains := []string{}
+	for domain := range l.Policies {
+		domains = append(domains, domain)
+	}
+	return domains, nil
+}
+
+// HostnamesForDomain [interface Validator] retrieves the hostname policy for
+// a particular domain.
```

Comment:
The comment about what interface this satisfies is really helpful. I think I will adopt that practice...